workshops/2019-10-29_ODSC-WEST/max-fashion-mnist-wrapping-solution/README.md

```diff
@@ -89,14 +91,24 @@ $ git clone https://github.com/IBM/MAX-Skeleton.git
 
 ## Update Dockerfile
 
```

Comment:
Because the model training is optional, I think it would be more clear if we structured this section differently.
For example:

```
# Update Dockerfile

If you have trained your own model, follow the steps below to upload your model file ...
[UPLOAD STEPS]

If you have not trained your own model and want to use our model file for demo purposes, ...
[NEXT STEPS]
```

Do you think this is better @SSaishruthi? Other than this, it looks good and feel free to merge this PR

Reply:
@splovyt 

That's a good point. But my question is, 

1. Participants are not going to train the model so they won't be going through `upload steps`
2. If we are adding upload steps, we should tell them the process of uploading either in COS or similar storage which might increase the complexity.

I would opt for just telling them in a note (like we have now) that if they have their own trained model, get the location . of the model file and update here.

Let me know what you think

Reply:
> Participants are not going to train the model so they won't be going through upload steps

This is what I am trying to say. The way it's formulated right now looks like they need to upload something, and only later on it says that if you want to use our demo assets you can.  What I suggested was to tell the reader first that they only need to do the next step if they have trained their own model. 

To be fair, it's a very minor remark so I'd be more than happy to just merge this without change.

Reply:
@splovyt 

Oh. I thought `For demo purpose` will make it clear to the users.
I have updated the section now, can you please check?